app_dart/test/request_handlers/update_existing_flaky_issues_test_data.dart

```diff
@@ -203,66 +207,82 @@ https://flutter-dashboard.appspot.com/#/build?taskFilter=Linux%20ci_yaml_flutter
-    timeout: 60
-    properties:
-      add_recipes_cq: "true"
-      dependencies: >-
-        [
-          {"dependency": "android_sdk", "version": "version:29.0"},
-          {"dependency": "chrome_and_driver", "version": "version:84"},
-          {"dependency": "open_jdk", "version": "11"},
-          {"dependency": "xcode", "version": "13a233"},
-          {"dependency": "gems", "version": "v3.3.14"},
-          {"dependency": "goldctl", "version": "git_revision:3a77d0b12c697a840ca0c7705208e8622dc94603"}
-        ]
-      shard: build_tests
-      subshard: "1_4"
-      tags: >
-        ["framework","hostonly","shard"]
-    scheduler: luci
-''';
+final CiYaml testCiYaml = CiYaml(
+  slug: Config.flutterSlug,
+  branch: Config.defaultBranch(Config.flutterSlug),
+  config: pb.SchedulerConfig(
+    enabledBranches: <String>[
+      Config.defaultBranch(Config.flutterSlug),
+    ],
+    targets: <pb.Target>[
+      pb.Target(
+        name: 'Mac_android android_semantics_integration_test',
+        scheduler: pb.SchedulerSystem.luci,
+        presubmit: false,
+        properties: <String, String>{
+          'tags': jsonEncode(['devicelab'])
+        },
+      ),
+      pb.Target(
+          name: 'Mac_android ignore_myflakiness',
+          scheduler: pb.SchedulerSystem.luci,
+          presubmit: false,
+          properties: <String, String>{
+            'ignore_flakiness': 'true',
+            'tags': jsonEncode(['devicelab']),
+          }),
+      pb.Target(
+        name: 'Linux ci_yaml flutter roller',
+        scheduler: pb.SchedulerSystem.luci,
+        bringup: true,
+        timeout: 30,
+        runIf: ['.ci.yaml'],
+        recipe: 'infra/ci_yaml',
+        properties: <String, String>{
+          'tags': jsonEncode(["framework", "hostonly", "shard"]),
+        },
+      ),
+      pb.Target(
+        name: 'Mac build_tests_1_4',
+        scheduler: pb.SchedulerSystem.luci,
+        recipe: 'flutter/flutter_drone',
+        timeout: 60,
+        properties: <String, String>{
+          'add_recipes_cq': 'true',
+          'shard': 'build_tests',
+          'subshard': '1_4',
+          'tags': jsonEncode(["framework", "hostonly", "shard"]),
+          'dependencies': jsonEncode([
+            {
+              'dependency': 'android_sdk',
+              'version': 'version:29.0',
+            },
+            {
+              'dependency': 'chrome_and_driver',
+              'version': 'version:84',
+            },
+            {
+              'dependency': 'xcode',
+              'version': '13a233',
+            },
+            {
+              'dependency': 'open_jdk',
+              'version': '11',
+            },
+            {
+              'dependency': 'gems',
+              'version': 'v3.3.14',
+            },
+            {
+              'dependency': 'goldctl',
+              'version': 'git_revision:3a77d0b12c697a840ca0c7705208e8622dc94603',
+            },
+          ]),
+        },
+      ),
+    ],
+  ),
+);
```

Comment:
Can the integration test also be updated to execute the bin/ script?

Reply:
Yes but if you don't mind explaining, what is the difference?

Reply:
Oh.. I understand now. I'll make the fix.

Reply:
For posterity, we can ensure better test coverage if we fix it so:

integration_test -> bin -> lib

Whereas today it's doing

bin -> lib
integration_test -> lib

Today has no test coverage for bin which leads to these breakages landing

Reply:
Done. I have updated the test to run so that we are kicking off code in bin only which calls into lib.